data/json/monsters/zed_wooden.json

```diff
@@ -0,0 +1,153 @@
+      { "id": "smash", "throw_strength": 36, "cooldown": 30 }
+    ],
+    "death_drops": "default_zombie_clothes",
+    "upgrades": { "half_life": 42, "into": "mon_zombie_wood_2" },
+    "burn_into": "mon_zombie_scorched",
+    "flags": [ "SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "REVIVES", "PUSH_MON", "FILTHY" ],
+    "armor": { "bash": 30, "cut": 36, "bullet": 25, "electric": 7 }
+  },
+  {
+    "id": "mon_zombie_wood_2",
+    "type": "MONSTER",
+    "name": { "str": "wooden hulk" },
+    "description": "This shambling mass looks almost as if a tree is chasing you.  Thick bark covers the entire body, dripping black ooze like sap from a wounded tree.  With arms that look like castle gates, it slams the ground and slowly but relentlessly lurches towards you.",
+    "looks_like": "mon_zombie_hulk",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "volume": "650 L",
+    "weight": "225 kg",
+    "hp": 350,
+    "speed": 65,
+    "material": [ "flesh", "wood" ],
+    "symbol": "Z",
+    "color": "brown",
+    "scents_tracked": [ "sc_human", "sc_fetid" ],
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 3,
+    "melee_dice_sides": 12,
+    "melee_damage": [ { "damage_type": "bash", "amount": 8 } ],
+    "dodge": 1,
+    "weakpoint_sets": [ "wps_humanoid_body", "wps_natural_armor" ],
+    "families": [ "prof_intro_biology", "prof_wp_zombie", "prof_wp_nat_armored", "prof_wp_hulk" ],
+    "bleed_rate": 0,
+    "vision_day": 50,
+    "vision_night": 4,
+    "harvest": "zombie_wood",
+    "special_attacks": [
+      { "id": "slam", "cooldown": 10, "damage_max_instance": [ { "damage_type": "bash", "amount": 40 } ] },
+      { "id": "longswipe" },
+      { "id": "smash", "throw_strength": 72 }
+    ],
+    "death_drops": "mon_zombie_wood_death_drops",
+    "burn_into": "mon_zombie_scorched",
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "DESTROYS",
+      "POISON",
+      "NO_BREATHE",
+      "REVIVES",
+      "PUSH_MON",
+      "PUSH_VEH",
+      "FILTHY"
+    ],
+    "armor": { "bash": 35, "cut": 40, "bullet": 30, "electric": 8 }
+  },
+  {
+    "id": "mon_zombie_wood_death_drops",
+    "type": "item_group",
+    "subtype": "distribution",
+    "entries": [ { "group": "default_zombie_clothes", "prob": 80 }, { "group": "corpse_wooden_zombie", "prob": 70 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "corpse_wooden_zombie",
+    "items": [
+      { "item": "twig", "prob": 100, "count-min": 5, "count-max": 20 },
+      [ "stick", 80 ],
+      [ "acorns", 20 ],
+      [ "pinecone", 20 ],
+      { "item": "withered", "prob": 80, "count-min": 1, "count-max": 10 },
+      { "item": "plant_fibre", "prob": 80, "count-min": 1, "count-max": 10 },
+      [ "pine_bough", 10 ],
+      [ "splinter", 90 ],
+      [ "tinder", 50 ]
+    ]
+  }
```

Comment:
These don't belong in the monster file 😛
Into monsterdrops/zombie they go

Reply:
TY, first monster I've made!